client/my-sites/sidebar/appearanceMenu.jsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { includes } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import SidebarButton from 'layout/sidebar/button';
+import SidebarHeading from 'layout/sidebar/heading';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import { canCurrentUser } from 'state/current-user/selectors';
+import { getSelectedSite } from 'state/ui/selectors';
+import { getCustomizeUrl } from 'my-sites/themes/helpers';
+
+class AppearanceMenu extends Component {
+
+	static propTypes = {
+		path: PropTypes.string,
+		site: PropTypes.object.isRequired,
+		sites: PropTypes.object.isRequired,
+	}
+
+	isSingle() {
```

Comment:
I'm thinking isSingle could probably be defined when mapping state to props to avoid having to call it a number of times...